shared/macros-bash.jinja

```diff
@@ -156,6 +156,170 @@ fi
 dconf update
 {{%- endmacro -%}}
 
+{{#
+# Macro to enable or disable a particular service.
+#
+# Examples:
+#
+# bash_service_command("enable", "bluetooth")
+# bash_service_command("disable", "bluetooth.service")
+# bash_service_command("disable", "rsh.socket", xinetd="rsh")
+#}}
+{{%- macro bash_service_command(service_state, service, xinetd="") -%}}
+local service_state="{{{ service_state }}}"
```

Comment:
Heads up - this work is about getting rid of functions, so the `local` keyword should typically be removed.